.github/workflows/rebase.yml

```
@@ -0,0 +1,24 @@
+name: Automatic Rebase
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  rebase:
+    name: Rebase
+    runs-on: ubuntu-latest
+    if: >-
+      github.event.issue.pull_request != '' && 
+      contains(github.event.comment.body, '/rebase') &&
+      github.event.comment.author_association == 'MEMBER'
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v3
```

There is a newer version of checkout available
```suggestion
        uses: actions/checkout@v4
```